<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilePropertyInfo</class>
 <widget class="QDialog" name="FilePropertyInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>300</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>文件名: </string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="file_name">
     <property name="text">
      <string>File Name</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>拥有者: </string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="account">
     <property name="text">
      <string>Owner</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string> 上传时间: </string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="create_time">
     <property name="text">
      <string>Create Time</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="maximumSize">
      <size>
       <width>70</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>文件大小: </string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="file_size">
     <property name="text">
      <string>File Size</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>分享状态: </string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="share_status">
     <property name="text">
      <string>Share Status</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>下载地址: </string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="download_url">
     <property name="text">
      <string>Download Path</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>下载量: </string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="downloads">
     <property name="text">
      <string>Downloads</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
